ob_v3p0/errata/errata.md

```diff
@@ -2,7 +2,28 @@
 
 ### Context file
 
-The context file for Open Badges 3.0 follows a versioning as a result of https://github.com/1EdTech/openbadges-specification/issues/497. All changes to this file that may lead to invalid signatures and errors must lead to a new version of the file. This section recaps all the changes made to this file.
+The context file for Open Badges 3.0 follows a versioning as a result of https://github.com/1EdTech/openbadges-specification/issues/497. All changes to this file that may lead to invalid signatures and errors (breaking changes) must lead to a new version of the file.
+
+#### What constitutes a breaking change
+
+The policy used to determine what constitutes breaking changes to the specification, and therefore to the context file, can be resumed as:
+
+- Adding new terms to the model/context *is not* a breaking change.
+- Changing term mappings or removing terms *is* a breaking change.
+
+This policy implies that a downstream consumer must be willing to be forgiving on the data they receive: they must be willing/capable of ignoring terms they don't currently understand.
+
+An example for this could be the change *"add the term identifier to the CLR context where it may appear in nodes of type ClrSubject"*. Let's look at how an issuer and a verifier of a ClrCredential might need to understand or react to this change:
```

Comment:
Typically we would use some way to call out the name of the term to make it explicit. I suggest something like this:
```
An example for this could be the change *"add the term 'identifier' to the CLR context where
it may appear in nodes of type 'ClrSubject'"*. Let's look at how an issuer and a verifier of a
ClrCredential might need to understand or react to this change:
```

Then, in the following text where we want to call out the term by name we also would use 'identifier' or "identifier" (we'd favour the double-quotes, except where we are quoting the term _inside_ another set of double quotes; then we'd use single quotes there, as in my suggested example above).